pkg/controller/install/webhook.go

```diff
@@ -101,6 +102,12 @@ func (i *StrategyDeploymentInstaller) createOrUpdateMutatingWebhook(ogNamespacel
 			log.Errorf("Webhooks: Error creating MutatingWebhookConfiguration: %v", err)
 			return err
 		}
+
+		if len(desc.ConversionCRDs) != 0 && isSingletonOperator(i) {
```

Comment:
Maybe a comment here about why `isSingletonOperator` is being used? Assuming this is because we are only allowing conversion webhooks on global operator installs?

Reply:
Yes, I'll add that to the comment.